src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -250,6 +250,15 @@ void setOutputPath(String pathString) {
       defaultValue = "NORMAL")
   private RunMode mode;
 
+  @Option(
+      names = "--need-querytext",
```

Comment:
You might want to add `negatable=true` so you can define `--no-need-querytext`.

Reply:
Great idea. On the other hand, picocli has a quirky decision regarding negatable flags with default true value (https://picocli.info/#_negatable_options) so I had to change the default behavior definition a bit.